students/thorn/lesson06/mailroom_6_test.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+import mailroom_6
+
+# TODO: mocks
+
+class mailroom_6_unit_test(unittest.TestCase):
+    ''' Sets up test information. '''
+    donors = {
+    'Tom Horn': [599.23, 1000.00],
+    'Theo Hartwell': [0.01, 0.01, 0.1],
+    'Bailey Kimmitt': [8723.22, 27167.22, 91817.66],
+    'Paul Hubbell': [90012.32, 2312.24],
+    'David Beckham': [1817266.11, 123123.66, 111335.112]
+    }   
```

Comment:
You can use `unittest.TestCase.setUp` method for defining shared data and set up logic between test methods. See [this section](https://docs.python.org/3/library/unittest.html#organizing-test-code) of the unittest docs for how to organize test code.